models/consistency.go

```diff
@@ -39,7 +39,7 @@ func CheckConsistencyFor(t *testing.T, beansToCheck ...interface{}) {
 		ptrToSliceValue := reflect.New(sliceType)
 		ptrToSliceValue.Elem().Set(sliceValue)
 
-		assert.NoError(t, x.Where(bean).Find(ptrToSliceValue.Interface()))
+		assert.NoError(t, x.Table(bean).Find(ptrToSliceValue.Interface()))
```

Comment:
This is relevant?

Reply:
I will open a new PR to instead this one. It's unrelated but should be a bug.

Reply:
@lunny can you reverence the PR to tis bugfix?

Reply:
for the log -> #7271